Formula/lesspipe.rb

```diff
@@ -16,7 +17,12 @@ class Lesspipe < Formula
   end
 
   def install
-    system "./configure", "--prefix=#{prefix}"
+    args = %W[
+      --prefix=#{prefix}
+    ]
```

Comment:
```suggestion
    args = *std_configure_args
```

Reply:
This gives an error as the the script exits on the unknown arguments `disable-debug`, `disable-dependency-tracking` and `libdir`. Is there another way to give just the single `prefix` argument?

Reply:
No, if it's not a real configure script this is fine.